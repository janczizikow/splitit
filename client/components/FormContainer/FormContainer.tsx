import styled from "@emotion/styled";

const FormContainer = styled.div`
  padding: 1rem;
  margin: auto;
  width: 100%;
  max-width: 432px;
`;

FormContainer.displayName = "FormContainer";

export default FormContainer;
