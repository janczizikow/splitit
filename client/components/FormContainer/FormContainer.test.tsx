import { mount } from "enzyme";
import toJson from "enzyme-to-json";
import FormContainer from "./FormContainer";

describe("<FormContainer />", () => {
  it("matches snapshot", () => {
    const wrapper = mount(<FormContainer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
