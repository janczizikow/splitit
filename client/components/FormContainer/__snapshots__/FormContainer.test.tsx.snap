// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormContainer /> matches snapshot 1`] = `
.emotion-0 {
  padding: 0 1rem;
  margin: 0 auto;
  width: 100%;
  max-width: 432px;
}

<FormContainer>
  <div
    className="emotion-0"
  />
</FormContainer>
`;
